charts/fybrik/files/taxonomy/taxonomy.json

```diff
@@ -294,6 +300,19 @@
         "spreadsheet_id"
       ]
     },
+    "https": {
+      "type": "object",
+      "description": "Connection information for accessing data via https",
+      "properties": {
+        "url": {
+          "type": "string",
+          "description": "The URL path to access the file."
+        }
+      },
+      "required": [
+        "url"
+      ]
+    },
```

Comment:
Can this connection be used for storing data? Does it make sense to define a storage account for this type? If so, perhaps to define more fields, like domain/base_url, folder and file?

Reply:
@shlomitk1 rethinking about that, there are libs such as `url.Parse` which could parse the url and provide the fields you mentioned, so we do not need to specify them explicitly but rather provide the url. Does it make sense?

Reply:
ok